python/py-sacremoses/Portfile

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+PortGroup           github 1.0
+
+github.setup        alvations sacremoses 99102ce
+version             0.0.35
+revision            0
+name                py-sacremoses
+
+license             LGPL-2.1+
+maintainers         nomaintainer
+platforms           darwin
+
+description         Python port of Moses tokenizer, truecaser and normalizer
+
+long_description    ${description}
+
+homepage            https://github.com/alvations/sacremoses
+
+python.versions     37 38
+
+checksums           rmd160  fc932022c13a1028e5d854bc5513f18d7453f291 \
+                    sha256  6ea400212c2c8605fe80afec888a0d00f40c475d88a996294a04c719a42b2b4e \
+                    size    879729
+
+if {${name} ne ${subport}} {
+    # see https://github.com/alvations/sacremoses/blob/master/setup.py
+    depends_lib-append \
+                    port:py${python.version}-setuptools
+    depends_run-append \
+                    port:py${python.version}-click \
+                    port:py${python.version}-joblib \
+                    port:py${python.version}-six \
+                    port:py${python.version}-tqdm
+}
```

Comment:
please add ```livecheck.type none``` within the ```if```-block, we only do the livecheck for the ```py-port``` not its subports.

Reply:
@reneeotten Thanks. But doesn't that mean it should appear in a `if {${name} eq ${subport}} ` block?

Reply:
correct, that's also what I said in my review comment; I might have clicked a line too low ;)

Reply:
@reneeotten Would you please double check that the `livecheck` is in the right block? It’s now in the `ne` block.

Reply:
I am sorry, I didn't read your previous question correctly.... It is correct right now, it should be in the ```if {${name} ne ${subport}}```, which means that if it will be "off" for ``pyXY-name``` and "on" for ```py-name```. You can convince yourself about this by doing ```port -dv livecheck py-sacremoses``` and ```port -dv livecheck py37-sacremoses``` and check the output